code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -134,6 +152,22 @@
 		var/datum/action/A=X
 		A.UpdateButtonIcon()
 
+/obj/item/clothing/head/helmet/space/plasmaman/AltClick(mob/user)
+	..()
+	. = COMSIG_CLICK_ALT
+	if(!attached_hat)
+		return
+	if(user.put_in_active_hand(attached_hat))
+		to_chat(user, "<span class='notice'>You removed the [attached_hat.name] from the helmet!</span>")
+		attached_hat = null
+		update_icon()
+	else if(user.put_in_inactive_hand(attached_hat))
+		to_chat(user, "<span class='notice'>You removed the [attached_hat.name] from the helmet!</span>")
+		attached_hat = null
+		update_icon()
+	else
+		to_chat(user, "<span class='notice'>You can't remove the [attached_hat.name] from the helmet because your hands are full!</span>")
```

Comment:
alternatively could drop it on the floor if someone thinks that's better

Reply:
i personally think it's fine not dropping it

Reply:
i'd say i prefer it not dropping either way, but others may think different

Reply:
wait, i just thought of something. what if they don't have hands? it'd still say their hands are full, wouldn't it?

Reply:
Well, if you don't have hands, you can't really take off anything so taking off the hat would be pretty dumb. Now I think that this way is better.

Reply:
i think it would still say "your hands are full" if you had no hands tho, and I can imagine some player thinking or saying "Pepega I have no hands OMEGALUL"

Reply:
https://youtu.be/yKysacndecI